christfellowshipchurch/web-app-v2
# Increase discover cards limit

### About
The Discover category pages had a limit of 21 cards to be displayed, that limit is now increased to 50, which resolves this issue.

### Test Instructions
1. Open the Discover page 
2. Open Devotionals 
3. Open "21 Days of Prayer And Fasting - 2021 Edition"
4. There should be 22 cards now, before there was a limit of 21 cards so only 21 cards were displayed 

### Closes Tickets
[CFDP-1554]


[CFDP-1554]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1554
